Tools/AP_Periph/APM_Config.h

```diff
@@ -0,0 +1,17 @@
+// User specific config file.
```

Comment:
I named it like that because of the same file in ArduCopter. I was trying to keep changes minimal because I am sure a lot of users are already familiar with this.

Reply:
does user hook init work with rover?